js/renovation/ui/button.tsx

```diff
@@ -235,13 +235,13 @@ export class Button extends JSXComponent(ButtonProps) {
 
```

Comment:
maybe it's better to get the props this way:
```
   const  text = this.props.text ?? '';
   const icon = this.props.icon ?? '';
```
to avoid changes below?

We can use this approach everywhere now and remove this additional type check after the question about possible `undefined` assignment is solved.

@churkin  what do u think?